include/swift/AST/Decl.h

```diff
@@ -4577,7 +4574,7 @@ class ParamDecl : public VarDecl {
   ParameterTypeFlags getParameterFlags() const;
   
   SourceLoc getSpecifierLoc() const { return SpecifierLoc; }
-
+    
```

Comment:
Can you try to remove whitespace-only changes, especially ones that add whitespace, and even more so what git considers 'whitespace errors'?

You can check for whitespace errors with `git diff --check` (or `git diff --staged --check` for staged changes).